src/main/java/org/parabot/core/asm/wrappers/Getter.java

```diff
@@ -33,12 +34,13 @@
      */
     public Getter(final String into, final String fieldLocation, final String fieldNode,
                   final String methodName, final String returnDesc, final boolean staticMethod, final long multiplier,
-                  final String fieldDesc) {
+                  final String fieldDesc) throws FieldNotFoundException {
+
         this.into = ASMUtils.getClass(into);
         this.fieldLocation = ASMUtils.getClass(fieldLocation);
         this.fieldNode = ASMUtils.getField(ASMUtils.getClass(fieldLocation), fieldNode, fieldDesc);
         this.methodName = methodName;
-        this.returnDesc = returnDesc == null ? this.fieldNode.desc : returnDesc;
+        this.returnDesc = returnDesc == null ? null : returnDesc;
```

Comment:
OK to be null, `addGetterAdapter` does `this.returnDesc = returnDesc == null ? fieldNode.desc : returnDesc;`